packages/components/organisms/f-footer/src/components/LinkList.vue

```diff
@@ -88,32 +86,35 @@ export default {
             return this.currentScreenWidth <= 1024;
         }
     },
-
     mounted () {
         this.currentScreenWidth = windowServices.getWindowWidth();
         windowServices.addEvent('resize', this.onResize, 100);
 
-        if (this.isBelowWide) {
-            this.panelCollapsed = true;
-        }
+        this.setPanelCollasped();
     },
-
     destroyed () {
         windowServices.removeEvent('resize', this.onResize);
     },
-
     methods: {
+        setPanelCollasped () {
```

Comment:
Small thing – can you add JSDoc comments to any new methods added ([like this style](https://github.com/justeat/fozzie-components/blob/master/packages/components/molecules/f-searchbox/src/components/Form.vue#L410-L416)).

We're trying to add these to all methods going forward as well as go back and add them to existing methods (so if you feel like you can add them to the existing methods below too, that would be awesome too 👍 ).

Reply:
Also typo here, should be `collapsed`

Reply:
Oh no! Not again.